diff --git a/openair2/LAYER2/NR_MAC_gNB/gNB_scheduler_RA.c b/openair2/LAYER2/NR_MAC_gNB/gNB_scheduler_RA.c
index ecd2c2cb84..dc533f68b3 100644
--- a/openair2/LAYER2/NR_MAC_gNB/gNB_scheduler_RA.c
+++ b/openair2/LAYER2/NR_MAC_gNB/gNB_scheduler_RA.c
@@ -1485,9 +1485,7 @@ static void nr_generate_Msg2(module_id_t module_idP,
                               ? NR_RRC_SETUP_DELAY_MS + NR_RRC_BWP_SWITCHING_DELAY_MS
                               : NR_RRC_SETUP_DELAY_MS;
       int delay = nr_mac_get_reconfig_delay_slots(ra->DL_BWP.scs, delay_ms);
-      /* TODO check if the UE should apply configuration before sync */
-      interrupt_followup_action_t action = UE->reconfigCellGroup ? FOLLOW_INSYNC_RECONFIG : FOLLOW_INSYNC;
-      nr_mac_interrupt_ue_transmission(RC.nrmac[module_idP], UE, action, delay);
+      nr_mac_interrupt_ue_transmission(RC.nrmac[module_idP], UE, FOLLOW_INSYNC, delay);
     }
   }
 
